sass/base/_components.scss

```diff
@@ -30,10 +30,10 @@
 @import "components/media-block/media-block";
 @import "components/media/media";
 @import "components/meta-icons/meta-icons";
+@import "components/navigation/feature-nav";
```

Comment:
Hmmm do we want to move this before the main nav? There's a chance this might mess with cascaded styles as the feature nav inherits from the main nav...